presto-client/src/main/java/io/prestosql/client/QueryResults.java

```diff
@@ -79,7 +80,7 @@ public QueryResults(
             URI infoUri,
             URI partialCancelUri,
             URI nextUri,
-            List<Column> columns,
+            Optional<List<Column>> columns,
```

Comment:
Either revert the change here, or update `data` to be Optional as well

Reply:
I'll revert.